packages/web-client/app/components/card-pay/card-picker/index.css

```diff
@@ -0,0 +1,91 @@
+.card-picker-dropdown {
+  --default-field-height: 2.5rem;
+  --default-field-width: 24rem;
+  --options-menu-height: 33.2rem;
+
+  position: relative;
+  z-index: 10;
+  max-width: 100%;
+  color: var(--boxel-dark);
+  font: var(--boxel-font);
+  letter-spacing: var(--boxel-lsp-sm);
+}
+
+.card-picker-dropdown .ember-basic-dropdown-content-wormhole-origin {
+  position: absolute;
+}
+
+.card-picker-dropdown--selected .ember-basic-dropdown-content-wormhole-origin {
+  left: -100%;
+}
+
+.card-picker-dropdown > .ember-power-select-trigger {
+  min-height: var(--default-field-height);
+  width: var(--default-field-width);
+  padding: var(--boxel-sp-xs) var(--boxel-sp);
+  background: url('/images/icons/caret-dropdown.svg') right 1.125rem center/auto 0.5rem no-repeat;
+  background-color: var(--boxel-light);
+  border: 1px solid var(--boxel-purple-300);
+  border-radius: var(--boxel-border-radius);
+  transition: border-color var(--boxel-transition);
+}
+
+.card-picker-dropdown--selected > .ember-power-select-trigger {
+  width: auto;
+  background-size: 0.6rem auto;
+  border: none;
+}
+
+.card-picker-dropdown > .ember-power-select-trigger:hover {
+  border-color: var(--boxel-dark);
+  cursor: pointer;
+}
+
+.card-picker-dropdown .ember-basic-dropdown-content {
+  max-width: 100%;
+  height: 100%;
+  width: var(--default-field-width);
+  max-height: var(--options-menu-height);
+  background-color: var(--boxel-light);
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  border-radius: var(--boxel-border-radius);
+  box-shadow: 0 15px 30px rgba(0, 0, 0, 0.25);
+  overflow-y: auto;
+}
+
+.card-picker-dropdown .ember-basic-dropdown-content.ember-basic-dropdown--transitioning-in {
+  animation: drop-fade-below var(--boxel-transition);
```

Comment:
I think the animations are why the first card is autoselected. The autoselection happens only when you click at the bottom of the dropdown (~20px). If you change `drop-fade-below` to start at `-100px` instead, you will have the autoselect throughout the whole dropdown trigger.

The quick fix for this (and I think it's ok while animating), is to do `pointer-events: none` while the dropdown is opening.

Reply:
If this isn't great, there might be some PRs in ember-power-select that solve this, because others have faced similar issues.

Reply:
I tried the above suggestions, but still seeing the autoselection happen sometimes.

Reply:
I'm finding it hard to reproduce with the current change. Maybe you can create an issue for this?